release_notes.md

```diff
@@ -1 +1,2 @@
-
+## Bug fixes
+- Fix CallHttpAsync() to throw an HttpRequestException instead of a serialization exception if the target endpoint doesn't exist (#1718)
```

Comment:
Should we also mention that this is a breaking change, since we're changing the type of exception being thrown?

Reply:
That's a good point! I'll add it to the release notes.